core/src/main/java/com/arcrobotics/ftclib/command/PrintCommand.java

```diff
@@ -0,0 +1,27 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package com.arcrobotics.ftclib.command;
+
+/**
+ * A command that prints a string when initialized.
+ */
+public class PrintCommand extends InstantCommand {
+    /**
+     * Creates a new a PrintCommand.
+     *
+     * @param message the message to print
+     */
+    public PrintCommand(String message) {
+        super(() -> System.out.println(message));
```

Comment:
explain this?

Reply:
Huh wait this should've been replaced by `LogCatCommand`. Something must have messed up in the pull from HEAD.